src/hooks/useProjectPersistence.js

```diff
@@ -42,17 +42,26 @@ export const useProjectPersistence = ({
         const accessToken = user?.access_token;
         const params = { reactAppApiEndpoint, accessToken };
 
-        if (saveTriggered) {
+        if (saveTriggered || localStorage.getItem("awaitingSave")) {
```

Comment:
This looks great, where is `awaitingSave` set?

Reply:
it's set in standalone, when the login to save button is clicked - there's an incoming PR for that

Reply:
ah roger! I shall press the happy green button of joy!